tests/py-flux-test.py

```diff
@@ -0,0 +1,61 @@
+import unittest
+
+import numpy as np
+import pandas as pd
+from flux.thermal import PccThermalModel1D
+from flux.grids import setgrid
+
+class PyFluxTest(unittest.TestCase):
```

Comment:
rename to something like ThermalTestCase instead of PyFluxTest (other test cases will have names matching the module they test---e.g., in this case the TestCase is for flux.thermal)